subsys/dfu/fmfu_fdev/include/modem_update_types.h

```diff
@@ -0,0 +1,343 @@
+/*
```

Comment:
There's something weird with the generated files. Maybe regenerate?

Reply:
Re-generated